pyface/ui/qt4/util/image_helpers.py

```diff
@@ -99,3 +172,7 @@ def array_to_QImage(array):
     image._numpy_data = data
     return image
 
+
+# backwards compatible names
+array_to_QImage = array_to_image
+QImage_to_array = image_to_array
```

Comment:
do want to remove these in the next major release? i'm not sure if anyone outside of pyface uses them directly

Reply:
They might be being used in enable, but yes, they should probably go in the next major release.

Reply:
I've added a not to this effect, and will open an issue.

Reply:
thank you!

Reply:
and it doesn't look like they're used outside pyface - https://github.com/search?q=org%3Aenthought+QImage_to_array&type=code and https://github.com/search?q=org%3Aenthought+array_to_QImage&type=code